packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { Consumer } from 'xfc';
+import styles from './EmbeddedContentConsumer.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The source URL of the content to load.
+   */
+  src: PropTypes.string.isRequired,
+  /**
+   * Notifies the component that the container has been launched.
+   */
+  onLaunch: PropTypes.func,
+  /**
+   * Notifies the component that the container has been authorized.
+   */
+  onAuthorize: PropTypes.func,
+  /**
+   * The component can be configured with an authorization secret.
+   * secret - The authorization secret to be used if the embedded application does not know which domain to trust.
+   */
+  options: PropTypes.shape({
+    secret: PropTypes.string,
```

Comment:
[`resizeConfig`](https://github.com/cerner/xfc#iframe-resizing-config) needs to be added here as well.

Reply:
Due to the scope of the changes needed for the resizeConfig options we opted to introduce that on another pull request.

Reply:
What sort of changes would be required to include the resizeConfig?

Reply:
Here is the resizeConfig that needs to be supported based on the [xfc documentation](https://github.com/cerner/xfc#iframe-resizing-config). The resize config can be added to the options prop without any api changes. In addition, testing all possible resize config options would increase the size of this PR significantly.

Reply:
If I were to add the resizeConfig to the options would you prefer an object prop or a prop with the shape defined (see below). If I define a shape then I would have to restrict xfc to a specific version. @kafkahw @tbiethman thoughts?

Defining a shape will force a specific version of xfc.
```
options: PropTypes.shape({
  secret: PropTypes.string,
  resizeConfig: PropTypes.shape({
    scrolling: PropTypes.boolean,
    autoResizeWidth: PropTypes.boolean,
    fixedHeight: PropTypes.string,
    fixedWidth: PropTypes.string,
    heightCalculationMethod: PropTypes.oneOf([
      'bodyOffset',             // uses document.body.offsetHeight
      'bodyScroll',             // uses document.body.scrollHeight
      'documentElementOffset',  // uses document.documentElement.offsetHeight
      'documentElementScroll',  // uses document.documentElement.scrollHeight
      'max',                    // max of all of above options.
      'min',                    // min of all of above options.
	]),
    widthCalculationMethod: PropTypes.oneOf([
      'bodyOffset',             // uses document.body.offsetHeight
      'bodyScroll',             // uses document.body.scrollHeight
      'documentElementOffset',  // uses document.documentElement.offsetHeight
      'documentElementScroll',  // uses document.documentElement.scrollHeight
      'scroll',                 // max of bodyScroll and documentElementScroll.
      'max',                    // max of all of above options.
      'min',                    // min of all of above options.
	]),
	customCalculationMethod: PropTypes.func,
  }),
}),
```

Or, define options to be a generic object. This leaks that the component is a xfc wrapper.
```
options: PropTypes.object,
```